courses/free-courses-te.md

```diff
@@ -1,39 +1,222 @@
+* [MongoDB Database Tutorials for beginners in Telugu](https://www.youtube.com/playlist?list=PLeCpoxUq3Ehk9FDCQvswVhg5qrODqJ4Xp) - Sai Gopi
 
-* [Full Java Script Tutorials in Telugu - Telugu Web Guru](https://www.youtube.com/playlist?list=PLh6Yk2rpZu2KqDjTuU_qHr-tI_CHOkIsN) - telugu web guru (YouTube)
+
+### NextJS
+
+* [Next.js Crash Course in Telugu | Next.js in Telugu](https://www.youtube.com/watch?v=yqJlmkgroik) - Telugu Skillhub
+
+
+### NodeJS
+
+* [NodeJS Tutorial](https://www.youtube.com/watch?v=MY2Vxtfn5Tw) - Telugu Skillhub
+* [Node JS in Telugu | Node.js Tutorial for Beginners in Telugu](https://www.youtube.com/playlist?list=PLYnehuuSeAHtu27M2By66v6kJpF_oDR5I) - Know something!!!
+
+
+### PHP
+
+* [PHP Development Tutorial in Telugu - Learn in 24 Hours Part 1](https://www.youtube.com/watch?v=sX6g3zyPXkA) - Telugu Computer World
+* [PHP Development Tutorial in Telugu - Learn in 24 Hours Part 2](https://www.youtube.com/watch?v=zZ0QNRMxWkE) - Telugu Computer World
+* [PHP Tutorials](https://www.youtube.com/playlist?list=PLC2mgeYbYNm8TRhX27z6JG0DtR8FR_WuO) - Telugu Computer World
 
 
 ### Python
 
-* [Python in Telugu - Step by Step Tutorials](https://www.youtube.com/playlist?list=PLC2mgeYbYNm-3aTUq98pbmrA3P1_m-aJR) - Telugu Computer World (YouTube)
+* [Python in Telugu For Beginners - Complete Tutorial in 10 Hours](https://www.youtube.com/watch?v=fP9IvI4qu80) - Telugu Computer World
+* [Full Python Tutorial in Telugu | Telugu Web Guru](https://www.youtube.com/playlist?list=PLh6Yk2rpZu2JgeekeyLRQcwZsfLNbW8zQ) - Telugu Web Guru
 
 
 ### R
 
-* [R Programming Language Training Videos In telugu - R ట్రైనింగ్ వీడియోస్ ఇన్ తెలుగు R programming ట్రైనింగ్ వీడియోస్ ఇన్ తెలుగు](https://www.youtube.com/playlist?list=PLXx2-0oYp1LO9H8ciGQaTr6SN80dteTlc) - VLR Training (YouTube)
+* [R Programming Language Training Videos In telugu - R ట్రైనింగ్ వీడియోస్ ఇన్ తెలుగు R programming ట్రైనింగ్ వీడియోస్ ఇన్ తెలుగు](https://www.youtube.com/playlist?list=PLXx2-0oYp1LO9H8ciGQaTr6SN80dteTlc) - VLR Training
+* [What is R programming in telugu](https://www.youtube.com/watch?v=02CTDv02awE) - Python Life
+
+
+### ReactJS
+
+* [React JS In Telugu](https://www.youtube.com/watch?v=1r79Eqw6tfg) - Telugu Skillhub
+* [React JS In Telugu (Playlist)](https://www.youtube.com/playlist?list=PLWnZ0qt0PImVaDkDbF96dnRGO0_lXVLKf) - Telugu Skillhub
+
+
+### SQL
+
+* [SQL commands and PL/SQL programs complete in telugu - Oracle SQL-PL/SQL in telugu](https://www.youtube.com/watch?v=2XB5CddzEaM) - Edusoft Learning Systems - Learning Simplified
+* [Sql tutorials in telugu - sql video tutorials - sql tutorials for beginners telugu](https://www.youtube.com/playlist?list=PLXx2-0oYp1LPUXvjjriVMaMWALucsitR1) - VLR Training
+
+
+### Tailwind CSS
+
+* [How to make Responsive Website in 5 minutes using Tailwind CSS | TELUGU | UNQCODER](https://www.youtube.com/watch?v=YVaqIIlVr7s) - UNQ Coder
```

Comment:
This is only for 12 mins. We dont allow tutorials < 1hr in length. Please remove